README.md

```diff
@@ -18,6 +18,7 @@ integrated with many useful tools like PyTorch, sklearn, TensorFlow, etc.
 Optuna-Integration API reference is [here](https://optuna-integration.readthedocs.io/en/stable/reference/index.html).
 
 * [AllenNLP](https://optuna-integration.readthedocs.io/en/stable/reference/index.html#allennlp)  ([example](https://github.com/optuna/optuna-examples/tree/main/allennlp))
+* [Catalyst](https://optuna-integration.readthedocs.io/en/stable/reference/index.html#catalyst)
```

Comment:
Please add a link to https://github.com/optuna/optuna-examples/blob/main/pytorch/catalyst_simple.py.

Reply:
Thanks for the comment. I updated `README.md`.